spec/seeds/seeds_spec.rb

```diff
@@ -3,34 +3,32 @@
 
 def empty_ar_classes
   ar_classes = [
-    AllCasaAdmin,
-    CasaAdmin,
-    CasaCase,
-    CasaOrg,
-    CaseAssignment,
-    CaseContact,
-    ContactType,
-    ContactTypeGroup,
-    Supervisor,
-    SupervisorVolunteer,
-    User,
-    Volunteer
+      AllCasaAdmin,
+      CasaAdmin,
+      CasaCase,
+      CasaOrg,
+      CaseAssignment,
+      CaseContact,
+      ContactType,
+      ContactTypeGroup,
+      Supervisor,
+      SupervisorVolunteer,
+      User,
+      Volunteer
   ]
   ar_classes.select { |klass| klass.count == 0 }.map(&:name)
 end
 
 RSpec.describe "Seeds" do
-  ["development", "qa", "staging"].each do |environment|
-    describe "for environment: #{environment}" do
-      before do
-        Rails.application.load_tasks
-        allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new(environment))
-      end
+  describe "test development DB" do
+    before do
+      Rails.application.load_tasks
+      allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new('development'))
+    end
 
-      it "successfully populates all necessary tables" do
-        ActiveRecord::Tasks::DatabaseTasks.load_seed
-        expect(empty_ar_classes).to eq([])
-      end
+    it "successfully populates all necessary tables" do
+      ActiveRecord::Tasks::DatabaseTasks.load_seed
```

Comment:
At some point I would like to also add more asserts here- something like 
`expect(CasaOrg.first.supervisors.first.volunteers.first.cases.first.case_contacts.first.contact_type).not_to be_null` (that syntax might need some work) - basically something to make sure that everything in there is hooked together lately